### PR TITLE
Change license and add Mr. Nectar Palm

### PR DESCRIPTION
**Pull request by felix-hilden**
From #1: Here they are! I decided to have the licenses in our readme, because it seemed like a good place to add them because this repo is only for assets, and because RST rendering produces a quite nicely readable form.

### CODE REVIEW

**Comment by leycec** [comment]
*Excellent, excellent.* The branding of the bear begins!

> I decided to have the licenses in our readme...

This isn't a bad choice, actually. Simplifies matters. Simplicity matters.

That said, CC BY-NC-SA 4.0 is *possibly* not the best choice here. The NC clause in particular [makes that license incompatible with basically all existing open-source licenses](https://en.wikipedia.org/wiki/Creative_Commons_license#Adaptability) – most of which implicitly permit inclusion in for-profit products that otherwise abide by the terms of those licenses. NC prohibits that; thus, incompatible. This is bad.

The SA clause is also fairly restrictive. It's like the GPL of the art world, right? That's *not* necessarily a bad thing in the general case – but it is in this case. Why? Because these wondrous artefacts of your algorithmic muse *are also MIT-licensed,* a significantly less restrictive and more permissive license.

Altogether, these two observations mean that no sane artist or organization would choose to use these wondrous artefacts of your algorithmic muse under the highly restrictive CC BY-NC-SA 4.0 license. They'd just choose the MIT license, smoke a fat blunt shaped like a bear, and call it a day.

Sadly, that conclusion means that the CC BY-NC-SA 4.0 license had might as well not be there at all. It's the legal equivalent of a noop. Instead, might I suggest the **CC BY 4.0 license** as a useful alternative? The BY clause means that downstream consumers still have to credit you whenever they use your work, *just like in the MIT license.* That's all the MIT license grants, after all: **forced attribution.** Well, that and indemnity against lawsuit. Let's hope these logos never break someone's data centre, or the bears are in *real* trouble.

**tl;dr:**
CC BY-NC-SA 4.0 bad, because <<<<<<<< than MIT. CC BY 4.0 good, because ≈ to MIT.

*God help us if the lawyers ever get a hold of set theory.*

> Copyright (c) 2021 Beartype

Are you sure? You're happy to retain the copyright, too. I'm down with whatever you're down with, inventive artbro.

Lastly, bear <sup>*hyuk, hyuk*</sup> with me, but:

* Would you mind exporting these JPG files as PNG, too? *Gotta get dat sweet lossless action.*
* Do you have any SVG files or some such you'd like to commit as well? It might be nice for someone who is not me to be able to reconstruct the output JPG files and possibly even tweak, touch up, or otherwise molest the original vector artwork at some point. If not, no worries, of course!

You remain the Ultimate Arbiter of Art. Also, you have push access. Feel free to ignore everything I say and have your own way with this helpless repository.

**Comment by felix-hilden** [comment]
Oh, that's a good point! I wasn't really sure what we would want with a double license, but this makes perfect sense. Yeah I'm happy to take copyright, but I'm absolutely fine with just having attribution. I put *Beartype* there so that other assets wouldn't be automatically my copyright. Maybe I'll add a wording to the license that the copyrights of individual files are by the creators.

As to the file formats, I think you'll find that the files are already PNG 😅 And sadly I don't have any vector versions, which to be fair would be very convenient since I'm sure they allow for easier editing as well as arbitrary scaling. Alas, I haven't ascended to such levels in my graphical abilities :D Maybe if RST is able to render such images, a single image would suffice, but that's a thought for the next asset.

**Comment by felix-hilden** [comment]
I doubt JPG files would be useful, since the (mostly) black-and-white images are quite compressed already, and JPG would introduce artifacts, possibly visible particularly in the smaller version. And I'm sure you didn't mean it that way either :D So I'll merge this and get working on a PR on the other side!

**Comment by leycec** [comment]
> As to the file formats, I think you'll find that the files are already PNG :sweat_smile: 

Well, this is awkward. In my defense... I'm Canadian. Let's forget this ever happened. I've already forgotten, in fact.

> And sadly I don't have any vector versions...

Oh! No worries whatsoever. I extrapolated from the crisp vector-like line art that you were rendering bitmaps from original Inkscape- or Illustrator-based SVG files, but...

Well, this is awkward. This is wonderful as is. I didn't mean to imply that your work was deficient in some way, because it's the antonym of deficient: it's **supernal,** which means awesome but even better than awesome.

It's two bear paw thumbs up – way up! :feet: 

> I doubt JPG files would be useful...

Let's not do this. Let's also not ask why I misread `.png` as `.jpg` multiple times. I blame cross-country skiing for everything. First it took my free time; then, my brain. I pray those ice-encrusted skis don't come for my keyboard next.

Thanks for the dangerous flood of enthusiasm, Señor Bear Boss. Your work ethic makes me look bad, which I for one applaud. I have baked you this gluten-free pumpkin pie as a sign of my digital gratitude: :pie:

**Comment by felix-hilden** [comment]
Oh I didn't take it that way at all! No worries 😄 Making it as a vector didn't even occur to me, which it could have, saving lots of trouble, so I'll be taking that advice to heart 👍
